docs/app/pages/components/components-sections/tabs/tabs/tabs.component.html

```diff
@@ -86,11 +86,27 @@ <h4>{{ tab.title }}</h4>
 </uxd-api-properties>
 
 <uxd-api-properties tableTitle="Outputs">
+    <tr uxd-api-property name="activated" type="void">
+        This event is triggered when the tab is activated.
+    </tr>
     <tr uxd-api-property name="select" type="void">
         This event is triggered when the tab is selected.
+        <blockquote>
```

Comment:
There is a full stop missing at the end of the sentence. I'm also wondering is this a bit bulky to be inside the table?

![image](https://user-images.githubusercontent.com/20795331/63359641-fff67a80-c364-11e9-891e-d669e34ce2f7.png)

I think in particular the large bottom margin is slightly odd. Perhaps this would be fine if it was just a new paragraph?

![image](https://user-images.githubusercontent.com/20795331/63359724-21576680-c365-11e9-86a9-5deef4788758.png)

@donna-marijne thoughts?

Reply:
Yes, drop the blockquote. Actually, looking back, we've removed documentation for deprecated properties altogether. Can we do that, and post a console warning like we did with the masthead/page header?